mainwindow.cpp

```diff
@@ -638,6 +663,14 @@ QStringList MainWindow::saveAttackParameters()
     if (m_ui->checkBox_EnvironmentVar->isChecked()) {
         m_settings.setValue("environmentVariables", m_ui->lineEdit_EnvironmentVar->text());
     }
+    // Save unselected rows
+    QList<QVariant> unselectedRows;
```

Comment:
Let's make this into QList<int> so it's obvious what we store. QVariant is only to be used when really necessary and data type is indeed unknown.